pkg/internal/utils/utils.go

```diff
@@ -1,24 +1,7 @@
-package common
+package utils
 
 import "os"
 
-// NetpolError holds information about a single error/warning that occurred during running
-// connectivity analysis command (list or diff)
-type NetpolError interface {
-	IsFatal() bool
-	IsSevere() bool
-	Error() error
-	Location() string
-}
-
-// Ingress Controller const - the name and namespace of an ingress-controller pod
-const (
-	//  The actual ingress controller pod is usually unknown and not available in the input resources for the analysis.
-	// IngressPodName and IngressPodNamespace are used to represent that pod with those placeholder values for name and namespace.
-	IngressPodName      = "ingress-controller"
-	IngressPodNamespace = "ingress-controller-ns"
-)
-
 // formats supported for output of various commands
 const (
 	DefaultFormat = "txt"
```

Comment:
I think it is a bit confusing/weird that output formats are under `utils` pkg

Reply:
Maybe split to `pkg/internal/output` and `pkg/internal/testutils` ?

Reply:
done